.azurepipelines/azure-pipelines.yml

```diff
@@ -380,6 +396,14 @@ jobs:
         Build.Name:   "arlu"
         Build.Arch:   "-a x64"
         Build.Target: ""
+      BTLS_X86_DEBUG:
+        Build.Name:   "btls"
+        Build.Arch:   ""
+        Build.Target: ""
+      BTLS_X86_RELEASE:
+        Build.Name:   "btls"
+        Build.Arch:   ""
+        Build.Target: "-r"
```

Comment:
Does TTLS support X64 SBL? if yes had better have one build to cover X64 build.

Reply:
Good point @gdong1. BTL-S should essentially have parity with RPL-S so I added debug x64 builds. I also noticed x64 build was failing with OS Loader space overflow on ubuntu 20.04 and that this build wasn't included for RPL-S. I fixed the build error on both and added the RPL-S x64 debug build for ubuntu 20.04 (previously only windows was running x64).